plugins/blockvolume/hostvol_opts.go

```diff
@@ -1,34 +1,37 @@
 package blockvolume
 
 import (
+	"strconv"
+
+	"github.com/gluster/glusterd2/glusterd2/options"
 	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/size"
 
 	"github.com/pborman/uuid"
-	"github.com/spf13/viper"
 )
 
-// VolumeType represents a volume type
-type VolumeType string
-
 const (
-	// Replica represents a replica volume type
-	Replica VolumeType = "Replica"
+	defaultHostVolsize         = size.GiB * 5
+	defaultHostVolType         = "Replicate"
+	defaultHostVolReplicaCount = 3
+	hostVolautoCreate          = true
```

Comment:
Are two copies of default values necessary(also set in glusterd2/options/cluster.go)? can it be nil values or ?

Reply:
when we set values using cluster options, there might be a possibility of error. So in case of any error we can simply use these default values.